content/roadmap.md

```diff
@@ -0,0 +1,28 @@
+ Concrete high-level feature areas and changes.
```

Comment:
Don't we need to define this page somewhere under `config.yml` ?

Reply:
We do, I am just not sure where.